contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -381,29 +416,77 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
     /// @notice Called by StakingRouter to update the number of the validators of the given node
     /// operator that were requested to exit but failed to do so in the max allowed time
     ///
-    /// @param _nodeOperatorId Id of the node operator
-    /// @param _stuckValidatorsCount New number of stuck validators of the node operator
-    function updateStuckValidatorsCount(uint256 _nodeOperatorId, uint256 _stuckValidatorsCount) external {
-        _onlyExistedNodeOperator(_nodeOperatorId);
+    /// @param _nodeOperatorIds bytes packed array of the node operators id
+    /// @param _stuckValidatorsCounts bytes packed array of the new number of stuck validators for the node operators
+    function updateStuckValidatorsCount(
+        bytes _nodeOperatorIds,
+        bytes _stuckValidatorsCounts
+    )
+        external
+    {
         _auth(STAKING_ROUTER_ROLE);
-
-        _updateStuckValidatorsCount(_nodeOperatorId, uint64(_stuckValidatorsCount));
+        _requireValidReportData(_nodeOperatorIds.length % 8 == 0 && _stuckValidatorsCounts.length % 16 == 0);
```

Comment:
Maybe put calculation logic into the `_requireValidReportData()` method cause it's the same for updateExitedValidatorsCount